src/main/resources/assets/tconstruct/book/materials_and_you/en_us/modifiers/necrotic.json

```diff
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:necrotic",
+  "text": [
+    {
+      "text": "Death is on your side as you harvest the pure life from your foes for your own benefit."
+    }
+  ],
+  "effects": [
+    "Steals life from your enemies to heal you",
+    "Multiple levels, increases life steal amount per level",
```

Comment:
That is inaccurate. Additional levels increase the chance of life steal. At level 1, its a 10% chance, at level 5 its a 50% chance. I would update the first bullet point to be clear it adds a chance to steal life.